src/views/Restoration.vue

```diff
@@ -208,6 +222,14 @@ export default class Restoration extends Vue {
       // fetch entity snapshot
       const entitySnapshot = await this.fetchEntitySnapshot()
 
+      // set Applicant info in entitySnapshot
+      const stateFiling = entitySnapshot.businessInfo.stateFiling
+      const filing = stateFiling && await LegalServices.fetchFiling(stateFiling)
+      entitySnapshot.orgPersons = filing.restoration.parties as OrgPersonIF
+      entitySnapshot.orgPersons.forEach(o => {
+        o.officer.id = uuidv4()
+      })
+
```

Comment:
**fetchDirectors** or **fetchParties** Legal API endpoint return **business/entity** parties. 
For extension/conversion, correct parties need to be fetched from 'stateFiling' (applicant role) and set in entitySnapshot. 
Also, need to assign a randomly created 'id' using UUIDV4 as 'stateFiling' data does not return id for the orgPerson.